src/jubilation-line/index.jsx

```diff
@@ -0,0 +1,31 @@
+// @flow
+import React from 'react';
+import getContext from '../helpers/context';
+import Animation from '../jubilation-animation';
+
+type Props = { x1?: number, x2?: number, y1?: number, y2?: number};
```

Comment:
Reasonable.  I was working off of https://github.com/coreyflynn/jubilation/blob/master/src/jubilation-point/index.jsx#L6, which marks x and y as optional.

Will change for `Line`